From 025ca1c1bd9848bf12bff72b80b8d05d8e24968d Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Mon, 9 Jan 2023 20:25:43 +0100
Subject: [PATCH 2/2] Add setuptools to dependencies

pyproject.toml:
Pkg_resources is used in solid/solidpython.py, hence this project
depends on setuptools until the use of pkg_resources is replaced with
something else.
---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index 1ab7617..8b31bb7 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -38,6 +38,7 @@ euclid3 = "^0.1.0"
 pypng = "^0.0.19"
 PrettyTable = "=0.7.2"
 ply = "^3.11"
+setuptools = ">=65.6.3"
 
 [tool.poetry.dev-dependencies]
 tox = "^tox 3.11"
-- 
2.39.0

